_posts/2018-09-13-september-office-hours.md

```diff
@@ -0,0 +1,87 @@
+---
+title: September Office Hours
+description: Multi-user OpenFaaS Cloud, Istio Service Mesh, and Stateless Microservices
+
+date: 2018-09-13
+image: /images/september-office-hours/team.jpg
+categories:
+  - office-hours
+  - community
+  - kubernetes
+author_staff_member: burton
+dark_background: true
+---
+
+This month's OpenFaaS office hours meeting talks about OpenFaaS Cloud as a multi-user platform for teams, how we've improved the ability to run stateless microservices managed as functions, and includes a demo of Istio service mesh integration.
+
+The OpenFaaS community holds regular office hours for anyone to join in as well as more focused discussions with project members
+
+Types of calls:
+* Office Hours (750 community members)
+  * Questions and Answers
+  * Project updates and direction
+* Members Call (15 members)
+  * Members of the Github organization
+  * Assign and track issues
+* Core Contributors (5 core members)
+  * Wide project vision
+  * Help with project direction
+
+> All of the office hours meetings are recorded and available for later viewing on YouTube.
+
+<iframe width="700" height="394" src="https://www.youtube.com/embed/IQCnHMn1wdI?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```

Comment:
Use the youtube include tag for this thats been used in other posts (have a look at the 2018-07-14-kubernetes-operator-crd.md for example).

Reply:
Very cool! Didn't realize there was a special tag/parameter. Updated